typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -132,11 +135,21 @@ export const ChainMetadataSchema = z.object({
               .describe(
                 'The maximum number of blocks that this RPC supports getting logs for.',
               ),
-            minBlockNumber: z.number().positive().optional(),
-            maxBlockAge: z.number().positive().optional(),
+            minBlockNumber: z
+              .number()
+              .positive()
+              .optional()
+              .describe('The minimum block number that this RPC supports.'),
+            maxBlockAge: z
+              .number()
+              .positive()
+              .optional()
+              .describe(
+                'The maximum number of blocks that this RPC supports getting logs for.',
```

Comment:
No sorry, this is the "Maximum difference from the latest block for which the RPC can query data"

Reply:
Is it actually in relation to the latest block (like minBlockNumber) or about the window of getting logs? I would have interpreted it closer to the `chunk` parameter

Reply:
The comment on `RpcPaginationOptions` seems to confirm that?

Reply:
I think you have it backwards. minBlockNumber is absolute and maxBlockAge is relative to latest

Reply:
That's what the comments say right? Maybe we're saying the same thing here

Reply:
Sorry yes I looked at the wrong line